### PR TITLE
Frontpage design fixes

### PR DESCRIPTION
**Pull request by ajobi**
This PR draft contains styling adjustments of the frontpage. The arrow icons are still missing (in discussion with designer).

### CODE REVIEW

File: resources/views/styleguide.blade.php [hunk 1]
```diff
@@ -119,18 +119,6 @@
 </div>
 
 <div class="row">
-<section id="langswitch-card" class="col m-1">
```
Comment (igor-kamil) [inline resources/views/styleguide.blade.php:122]: ok... I am up for solving styleguide other way...

File: tailwind.config.js [hunk 2]
```diff
@@ -59,11 +59,14 @@ module.exports = {
         },
         lineHeight: {
             none: '1',
-            tight: '1.25',
-            snug: '1.375',
-            normal: '1.5',
-            relaxed: '1.625',
-            loose: '2',
+            tight: '1.08',
```
Comment (igor-kamil) [inline tailwind.config.js:62]: wow! such a precision...  lubica will like it :)

File: resources/js/components/atoms/links/LinkArrow.vue [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <template>
-    <a :href="url" class="group block leading-loose flex items-center">
+    <a :href="url" class="group block leading-relaxed flex items-center hover:text-blue">
         <slot></slot>
-        <span class="transform group-hover:translate-x-2 ml-3 duration-200">
-            →
+        <span class="transform group-hover:translate-x-2 ml-2 duration-200">
+            {{ external ? '→' : '->' }}
```
Comment (igor-kamil) [inline resources/js/components/atoms/links/LinkArrow.vue:5]: now I see
Screenshot 2021-04-23 at 12 10 40

**Review by igor-kamil — APPROVED** [review]
LGTM
respect for taking care for all these tiny details. looks better.

File: tailwind.config.js (hunk 2 above)
Comment (ajobi) [reply tailwind.config.js:62]: Thanks. Should have been done like this in the initial setup already tbh :-D

**Pull request merged**